control-plane/stor-port/src/types/v0/transport/pool.rs

```diff
@@ -272,6 +272,32 @@ impl CreatePool {
     }
 }
 
+/// Create Pool Request
```

Comment:
`.` missing at the end? Also the field doc comments should be converted to the format?

Reply:
I'd need to change entire file... I'll do it

Reply:
Thanks